db/migrate/20210310134947_create_courses.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateCourses < ActiveRecord::Migration[6.1]
+  def change
+    create_table :courses do |t|
+      t.string :name, null: false
```

Comment:
![image](https://user-images.githubusercontent.com/470137/110919058-bdfca300-8313-11eb-87db-441c2fd9703f.png)

I think we should also capture `course_code` as well.

https://github.com/DFE-Digital/teacher-training-api/blob/1a3da87ece457e0699a727862af8b32b6fab14b3/db/schema.rb#L122

`course_code` unique per `provider`

https://qa.publish-teacher-training-courses.service.gov.uk/organisations/1GT/2021/courses

Reply:
Yep... Good shout 👍

Reply:
Ahhh thanks @defong ! I assume if there's already a course with that code for that provider, we should update the name?

Reply:
yes the name can change... the `course code`  will be unlikely to change...

Reply:
ok - added all the code + uniqueness things!

Reply:
nice one :+1: